examples/layerorder.js

```diff
@@ -0,0 +1,97 @@
+goog.provide('layerorder');
+
+goog.require('ngeo.DecorateLayer');
+goog.require('ngeo.mapDirective');
+goog.require('ngeo.sortableDirective');
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.MapQuest');
+goog.require('ol.source.TileWMS');
+
+
+/** @const **/
+var app = {};
+
+
+/** @type {!angular.Module} **/
+app.module = angular.module('app', ['ngeo']);
+
+
+
+/**
+ * @param {angular.Scope} $scope Scope.
+ * @param {ngeo.DecorateLayer} ngeoDecorateLayer Decorate layer service.
+ * @constructor
+ * @ngInject
+ */
+app.MainController = function($scope, ngeoDecorateLayer) {
+
+  /** @type {ol.layer.Tile} */
+  var boundaries = new ol.layer.Tile({
+    source: new ol.source.TileWMS({
+      url: 'http://demo.opengeo.org/geoserver/wms',
+      params: {'LAYERS': 'topp:tasmania_state_boundaries'},
+      serverType: 'geoserver'
+    }),
+    name: 'Boundaries'
```

Comment:
I am picky but please use `set` for user properties:

```js
boundaries.set('name', 'Boundaries');
```

Although using a `name` option in the constructor options works here because the example code is not compiled, it won't work in a compiled application.

See https://github.com/camptocamp/ngeo/blob/master/docs/developer-guide.md#custom-olobject-properties to understand the reason.